beast/observationmodel/noisemodel/generic_noisemodel.py

```diff
@@ -10,11 +10,13 @@
 can be achieved using the trunchen method.  The trunchen method
 requires significantly more complicated ASTs and many more of them.
 """
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
 
 import numpy as np
 import tables
 
-import toothpick
+from . import toothpick
```

Comment:
The idea for the generic_noisemodel was to provide a noisemodel that could be used by most projects.  No need for the same code to be in every project directory.
Keeping the toothpick model details separate allows for someone to make a custom noisemodel if they want.